core/Cargo.toml

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 [package]
-name = "comet"
+name = "datafusion-comet"
```

Comment:
There is already a `comet` crate, so I renamed this. I have reserved the crate: https://crates.io/crates/datafusion-comet